meld-compare@klangman/meld-compare@klangman.nemo_action.in

```diff
@@ -0,0 +1,8 @@
+[Nemo Action]
+_Name=Compare using Meld
+_Comment=Compare files or folders using Meld
+Exec=meld %F
+Icon-Name=meld
+Selection=notnone
```

Comment:
You can change this to `Selection=2,3` so that it will only show the context menu when two or three files are selected. One file is moot and more than three generate a silent failure.

Looks amazing as always! Thanks!

Reply:
I tested this and I thought it was working but I don't see the context menu item on three items. I need to look into the syntax.